finesse/hardware/data_file_writer.py

```diff
@@ -79,7 +79,7 @@ def _get_hot_bb_power() -> float:
         return float("nan")
 
     try:
-        return hot_bb.power * 100 / config.TC4820_MAX_POWER
+        return hot_bb.power
```

Comment:
Technically this is returning an `int`, not a `float` now. Not sure why `mypy` isn't complaining.

Reply:
Also, if `TC4820_MAX_POWER` isn't used anywhere outside of `tc4820.py` now, we should move it there.